storey/table.py

```diff
@@ -244,6 +288,151 @@ def __getitem__(self, key):
          """
         return self._get_static_attrs(key)
 
+    async def _flush_worker(self):
+        try:
+            while not self._terminated:
+                await asyncio.sleep(self._flush_interval_secs)
```

Comment:
This is probably not exactly the semantic we want. If interval is every 2 seconds, and it took 2 seconds or more to persist, then we don't want to sleep at all. And if it took 1 second to persist, we only want to sleep one more second. So we need to use last_flush as a reference point here. I believe this is how it's done in _Batching.

Reply:
i don't think this semantics is better.
if the interval is 2 secs and it took 2 or more secs to flush then we start flushing again? so 100% of our time is dedicated to flushing? it's not right.
also, we flush everything when timeout expires, so what should be the time of "last_flush", when we start flushing the first event? when we finished the last? there's the all in_flight logic, do we consider it in the last_flush time? 
imo it will make stuff confusing and error prone without actual benefit.

Reply:
If we are meant to flush every minute, and it takes one minute to flush, then yes, I'd expect it to be flushing continuously. If we're supposed to flush every 10 seconds, and it takes one minute to flush, it makes no sense to wait another 10 seconds when flushing already took too long relative to what was intended. Otherwise it's basically an arbitrary sleep between flushes.

Reply:
We decided that we can let it be as is for now and maybe reconsider in the future.